lib/src/base-styles/_nx-variables.scss

```diff
@@ -52,7 +52,7 @@
   --nx-border-default: 1px solid var(--nx-color-border);
   --nx-border-2x: 2px solid var(--nx-color-border);
   --nx-border-secondary: 1px solid var(--nx-color-border-secondary);
-  --nx-border-width-button: 1px;
+  --nx-border-width: 1px;
```

Comment:
Since all CSS variables are part of our public API, I think we should mark this variable(`--nx-border-width-button`) as deprecated and update their values to be aliases for the equivalent new variable.